CONTRIBUTING.md

```diff
@@ -0,0 +1,118 @@
+# Contributing
+
+## General Workflow
+
+1. Clone the repo.
+1. Cut a namespaced feature branch from origin master.
+1. Make commits to your feature branch.
+1. Pull Request
+    - If you want to start a conversation about the feature   you're working on, submit a pull request to origin   master with [WIP]  prefixed.
+    - If you want to submit a pull request to merge your feature branch's code into master, make sure you've updated your feature branch by merging master's changes.
```

Comment:
Care to reword to be more clear? perhaps:
`If you want to submit a pull request to merge your feature branch's code into master, first make sure your feature branch is up-to-date by merging master's changes into yours.`